File: JAVA/class0113/src/class0113/Car.java
package class0113;

public class Car {
	public static final String JIJUM = "동대구 영업소";
	private String model;
	private String company;
	private String color;
	private String price;
	
	public Car() {
		//생성자
		model = "K7";
		company = "기아";
		color = "블랙";
		price = "사천만원";
	}

	public Car(String model, String company, String color, String price) {
		this.model = model;
		this.company = company;
		this.color = color;
		this.price = price;
	}
	
	
	public String getModel() {
		return model;
	}

	public void setModel(String model) {
		this.model = model;
	}

	public String getPrice() {
		return price;
	}

	public void setPrice(String price) {
		this.price = price;
	}

	public void showCarInfo() {
		System.out.println("-----------------------------");
		System.out.println("영업소 : " + Car.JIJUM);
		System.out.println("-----------------------------");
		System.out.println("제조사 : " + company);
		System.out.println("모델 : " + model);
		System.out.println("가격 : " + price);
		System.out.println("색상 : " + color);
		System.out.println("-----------------------------");
	}
	
	
	
}

File: csharp/carApp_0208/carApp_0208/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace carApp_0208
{
    class Car
    {
        // 클래스 상수 변수
        public const string DAERI = "동대구 영업소";

        // 인스턴스 변수
        private string company;
        private string color;
        private string price;
        private string model;

        // 기본 생성자
        public Car()
        {
            company = "기아";
            color = "블랙";
            price = "사천만원";
            model = "K7";
        }

        // 오버로딩
        // 생성자
        public Car(string company, string color, string model, string price)
        {
            this.company = company;
            this.color = color;
            this.model = model;
            this.price = price;
        }

        // propfull + 탭
        public string Company
        {
            get { return company; }
            set { company = value; }
        }
        public string Model
        {
            get { return model; }
            set { model = value; }
        }
        public string Price
        {
            get { return price; }
            set { price = value; }
        }
        public string Color
        {
            get { return color; }
            set { color = value; }
        }

        public void showCarInfo()
        {
            Console.WriteLine("제조사 :" + company);
            Console.WriteLine("색상 :" + color);
            Console.WriteLine("모델 :" + model);
            Console.WriteLine("가격 :" + price);
            Console.WriteLine("영업소 :" + DAERI);
            Console.WriteLine("------------------------");

        }

    }
}
